python_scripts/cross_validation_ex_01.py

```diff
@@ -3,12 +3,12 @@
 #
 # The aim of this exercise is to make the following experiments:
 #
-# * train and test a support vector machine classifier through
+# * train and test a support vector machine classification model through
```

Comment:
I feel like you can use just "model" in all this file.

support vector machine classification model sounds a bit too long